src/NexusMods.App.UI/Pages/ModLibrary/FileOrigins/FileOriginsPageViewModel.cs

```diff
@@ -51,22 +94,34 @@ public class FileOriginsPageViewModel : APageViewModel<IFileOriginsPageViewModel
 
         var loadout = _conn.Db.Get<Loadout.Model>(LoadoutId.Value);
         var game = loadout.Installation.Game;
+        try
+        {
+            _gameDomain = loadout.Installation.Game.Domain;
+        }
+        catch (Exception)
+        {
+            _gameDomain = GameDomain.DefaultValue;
+        }
```

Comment:
Why try/catch?

Reply:
No specific reason. 
It was part of the old code which was moved over.
I'll remove it.